broker/src/main/java/io/camunda/zeebe/broker/system/partitions/ZeebePartition.java

```diff
@@ -39,15 +42,18 @@
         SnapshotReplicationListener {
 
   private static final Logger LOG = Loggers.SYSTEM_LOGGER;
-  private Role raftRole;
 
+  private static final StartupProcess<PartitionStartupContext> STARTUP_PROCESS =
+      new StartupProcess<PartitionStartupContext>(LOG, List.of());
```

Comment:
FYI. In a parallel PR, I have introduced new startup steps. It might introduce merge conflicts.